src/Base/NestedMarkupExtension.cs

```diff
@@ -326,8 +326,13 @@ public object UpdateNewValue(TargetPath targetPath)
                 // Get the own formatted output.
                 object output = FormatOutput(targetPath.EndPoint, info);
 
-                // Set the property of the target to the new value.
-                SetPropertyValue(output, info, false);
+                var target = targetPath.EndPoint.TargetObject as TriggerBase;
```

Comment:
> why you double that code? This check is already done in the internal Method SetPropertyValue

I've tried to understand the code in more details, and as far as I can tell you are correct, this check is not required since it's already performed in SetPropertyValue (at least for the cases I could trace).